tests/console/apache.pm

```diff
@@ -33,13 +36,24 @@ sub run {
     assert_script_run 'echo "index" > /srv/www/htdocs/index.html';
 
     # Ensure apache2 is installed and stopped
-    if (script_run('rpm -q apache2') == 0) {
-        systemctl('stop apache2');
+    if (is_sle("=12-SP5")) {
+        if ($apache2 eq 'apache2-tls13') {
+            zypper_call 'rm apache2';
```

Comment:
Why do we need to remove the package here?
It has been installed at https://duck-norris.qe.suse.de/tests/12473#

Reply:
Seems remove and install the same packages :

https://duck-norris.qe.suse.de/tests/12473#step/apache/6
https://duck-norris.qe.suse.de/tests/12473#step/apache/8

Reply:
Strictly speaking it's not necessary, but this is to ensure that the correct package is installed and I think it's good to leave it there for this reason.

Reply:
Fine, one more question, do we need to remove `apache2*-utils` at the same time?

Reply:
Perhaps, but I think we're better off in not doing the `zypper rm` in the first place. Let me try to tidy this block a bit ... Will poke you when done.

Reply:
I removed the whole block, now we just install the corresponding apache package. This would cause a package conflict in case a conflicting apache2 installation is already present, so we should be safe.

Let's add the safeguard only if necessary to keep the codebase tidy.